src/main/python/cfn_sphere/stack_configuration/__init__.py

```diff
@@ -9,22 +9,20 @@
 class Config(object):
```

Comment:
It's a bit too clever yes, but it does a lot of neat things in a very convenient way. You think it would help, to add some documentation to it?